component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPKeyGenerator.java

```diff
@@ -56,10 +56,16 @@
 		String encodedQRCodeURL;
 		String tenantAwareUsername = null;
 		Map<String, String> claims = new HashMap<>();
+		long timeStep;
```

Comment:
This class is not formatted

Reply:
addressed with https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/77/commits/9441e9b18f9e89e6d4184cf0beef797878c68e7d